MetadataMigration/src/main/java/org/opensearch/migrations/commands/Evaluate.java

```diff
@@ -24,7 +25,10 @@ public EvaluateResult execute(RootMetadataMigrationContext context) {
             var clusters = createClusters();
             evaluateResult.clusters(clusters);
 
-            var transformer = selectTransformer(clusters);
+            var transformer = new CompositeTransformer(
```

Comment:
Refactor this to have `selectTransformer` do the work so both subclasses don't need to